common/CMakeLists.txt

```diff
@@ -2,48 +2,11 @@
 #
 # provides the common library
 
-add_subdirectory(pb)
-
-set(common_SOURCES
-    abstract_deck_list_card_node.cpp
-    abstract_deck_list_node.cpp
-    debug_pb_message.cpp
-    deck_list_card_node.cpp
-    deck_list.cpp
-    expression.cpp
-    featureset.cpp
-    get_pb_extension.cpp
-    inner_deck_list_node.cpp
-    passwordhasher.cpp
-    rng_abstract.cpp
-    rng_sfmt.cpp
-    server/game/server_abstract_participant.cpp
-    server/game/server_arrow.cpp
-    server/game/server_arrowtarget.cpp
-    server/game/server_card.cpp
-    server/game/server_cardzone.cpp
-    server/game/server_counter.cpp
-    server/game/server_game.cpp
-    server/game/server_player.cpp
-    server/game/server_spectator.cpp
-    server/server.cpp
-    server/server_abstractuserinterface.cpp
-    server/server_database_interface.cpp
-    server/server_protocolhandler.cpp
-    server/server_remoteuserinterface.cpp
-    server/server_response_containers.cpp
-    server/server_room.cpp
-    serverinfo_user_container.cpp
-    sfmt/SFMT.c
-)
+set(common_SOURCES)
```

Comment:
there are no more sources, there is no more library left then? all that is in here is peglib.h which is header only, why not move it to utils? or to another place where we put stuff that's header only like trice_limits.h?